pkgs/applications/graphics/sane/backends/brscan4/default.nix

```diff
@@ -21,7 +21,8 @@ stdenv.mkDerivation rec {
       url = "https://download.brother.com/welcome/dlf006645/${pname}-${version}.amd64.deb";
       sha256 = "sha256-Gpr5456MCNpyam3g2qPo7S3aEZFMaUGR8bu7YmRY8xk=";
     };
-  }."${stdenv.hostPlatform.system}";
+  # specify a default so src can be evaluated on any platform
+  }."${stdenv.hostPlatform.system}" or { };
```

Comment:
fixing these seems out of scope of this PR and not required for meta repository, in my opinion. there are other similar uses in the source tree, eg: `outfox` and there aren't any checks to prevent PRs from adding a src which can not be evaluated on some systems.

i don't see what the downside is of just leaving these cases as is.

Reply:
outfox uses `throw`, which can be caught using tryEval.  a throw error doesn't cause ofborg-eval to fail, since there is a tryEval in `release-attrpaths-superset.nix`.

the main user of this field will be nixos-search, which requires `meta` to be fully evaluated and converted into json, although i suppose it's probably fine as long as that is possible on x86_64-linux.